templates/component/terraform/fogg.tf.tmpl

```diff
@@ -40,7 +40,7 @@
 
 
 terraform {
-  required_version = "~>{{ .TerraformVersion }}"
+  required_version = "={{ .TerraformVersion }}"
```

Comment:
is that a breaking change? does it matter?

Reply:
Breaking in theory, but not in practice if you use the makefiles.